lib/render.js

```diff
@@ -9,12 +9,38 @@ var options  = require('./options');
 
 exports.toDisk = function(model) {
   var opts = options.get();
+  _.forEach(opts.lint, function(options, name){
+    lint(name, options, model);
+  })
   _.forEach(opts.renderers, function(options, name) {
     render(name, options, model);
   });
   console.log('\nDone!\n');
 };
 
+function lint(name, options, model){
+  var instance = null;
+  try {
+    instance = require('./linters/' + name);
+  } catch(ex) {
+    console.error('Unsupported linter: ' + name);
+    return;
+  }
+  console.log('\n-> Linting ' + name);
+  var errors = instance.lint(model, options).errors
+
+  if(errors.length){
+    switch(options){
+      case 1:
+        console.warn('WARNING: ' + name + " found invalid samples: " + errors.join(", "))
+        break;
+      case 2:
+        throw(new Error(name + " found invalid samples: " + errors.join(", ")));
+        break;
```

Comment:
Minor comment, could we `console.error` here, and then do a `process.exit(1)` at the end if necessary, so all errors get printed at once?